cns/ipampool/monitor.go

```diff
@@ -49,8 +52,8 @@ type Monitor struct {
 	once        sync.Once
 }
 
-// Global Variables for Subnet and SubnetAddressSpace
-var subnet, subnetCIDR string
+// Global Variables for Subnet, Subnet Address Space and Pod Network ARM ID
+var subnet, subnetCIDR, subnetARMID string
```

Comment:
previously called `podnetARMID`, change one or the other to be consistent (i think calling it subnetARMID is better)